From: Carlos Maddela <e7appew@gmail.com>
Date: Wed, 2 Mar 2016 04:20:06 +1100
Subject: Stop libav compatibility links from being considered as genuine.

Description: Stop libav compatibility links from being considered as genuine.
 Don't register version numbers for compatibility links, so that
 youtube-dl stops complaining about avconv being outdated.
Author: Carlos Maddela <e7appew@gmail.com>
Origin: vendor
Bug: https://github.com/rg3/youtube-dl/pull/8720
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=798936
Last-Update: 2016-06-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 youtube_dl/postprocessor/ffmpeg.py | 9 +++++++--
 1 file changed, 7 insertions(+), 2 deletions(-)

diff --git a/youtube_dl/postprocessor/ffmpeg.py b/youtube_dl/postprocessor/ffmpeg.py
index fa99b0c..29beb48 100644
--- a/youtube_dl/postprocessor/ffmpeg.py
+++ b/youtube_dl/postprocessor/ffmpeg.py
@@ -98,11 +98,16 @@ class FFmpegPostProcessor(PostProcessor):
                 self._paths = dict(
                     (p, os.path.join(location, p)) for p in programs)
                 self._versions = dict(
-                    (p, get_exe_version(self._paths[p], args=['-version']))
+                    (p, get_exe_version(self._paths[p], args=['-version'],
+                                        version_re=r'%s\s+version\s+([-0-9._a-zA-Z]+)' % p,
+                                        unrecognized=False))
                     for p in programs)
         if self._versions is None:
             self._versions = dict(
-                (p, get_exe_version(p, args=['-version'])) for p in programs)
+                (p, get_exe_version(p, args=['-version'],
+                                    version_re=r'%s\s+version\s+([-0-9._a-zA-Z]+)' % p,
+                                    unrecognized=False))
+                for p in programs)
             self._paths = dict((p, p) for p in programs)
 
         if prefer_ffmpeg:
